src/Runner.php

```diff
@@ -29,8 +29,7 @@ public function __construct(Container $container = null)
         $commands_directory = __DIR__ . '/Commands';
         $top_namespace = 'Pantheon\Terminus\Commands';
         $this->commands = $this->getCommands(['path' => $commands_directory, 'namespace' => $top_namespace,]);
-        $this->runner = new RoboRunner();
-        $this->runner->setContainer($container);
+        $this->runner = new RoboRunner(null, null, $container);
```

Comment:
This doesn't look right to me. Do you have the latest Robo master branch?  Try `composer update`.

I changed this to `setContainer()` to avoid the `null, null` thing.